Yoast/javascript
# Add Italian stemmer

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] add an Italian stemmer and edit its syntax 

## Relevant technical choices:

* We deleted one non-existing suffix (`abilamente`) from the original stemmer.
* There are 6 suffixes that are either extremely rare or also non-existing (`-istà`, `-istì`, `-istè`, `-ià`, `-iè`, `-iì`). We left these in the stemmer for now but made a note. Including them doesn't any harm, but in the future we can research further whether we can exclude them.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` and make sure that all tests pass.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-374
